src/__experimental__/components/grouped-character/grouped-character.spec.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import GroupedCharacter from '.';
+
+const mountComponent = props => mount(<GroupedCharacter { ...props } />);
+
+describe('GroupedCharacter', () => {
+  const basicGroupConfig = [2, 2, 4],
+      separator = '-',
+      valueString = '12345678';
+  let instance, input, onChange;
+
+  describe('functionality', () => {
+    beforeEach(() => {
+      onChange = jest.fn();
+
+      instance = mountComponent({
+        separator, groups: basicGroupConfig, value: valueString, onChange
+      });
+      input = instance.find('input');
+    });
+
+    it('takes configuration for how text should be grouped', () => {
+      expect(input.props().value).toEqual('12-34-5678');
+    });
+    it('emits an unformatted string', () => {
+      input.simulate('change', { target: { value: '123456' } });
+      expect(onChange).toHaveBeenCalledWith({ target: { value: '123456' } });
+    });
```

Comment:
this test is redundant

Reply:
`onChange` is the callback supplied by the dev, so this test asserts that the value supplied to that callback is the raw string, rather than a string containing separators.